README.md

```diff
@@ -88,7 +88,7 @@ OPTIONS:
    --guid              Display raw source GUIDs with no source Names. Incompativle with 'source' and 'source-type' for --sort-by. Incompatible with 'application' for --source-type
    --noise             Fetch and display the rate of envelopes per minute for the last minute. WARNING: This is slow...
    --sort-by           Sort by specified column. Available: 'source-id', 'source', 'source-type', 'count', 'expired', 'cache-duration', and 'rate'.
-   --source-type       Source type of information to show. Available: 'all', 'application', and 'platform'.
+   --source-type       "Source type of information to show. Available: 'all', 'application', 'service', 'platform', and 'unknown'. Excludes unknown sources unless 'all' or 'unknown' is selected, or `--guid` is used. To receive information on platform or unknown source id's, you must have the doppler.firehose, or logs.admin scope."
```

Comment:
I don't think that the `"` chars should be included at the beginning and end of this message.

Reply:
fixed